collect_app/src/main/java/org/odk/collect/android/formmanagement/ServerFormDetails.java

```diff
@@ -24,20 +24,16 @@
 
     private final String formName;
     private final String downloadUrl;
-    private final String manifestUrl;
     private final String formID;
     private final String formVersion;
     private final String hash;
     private final boolean isNotOnDevice;
     private final boolean isUpdated;
     private final ManifestFile manifest;
 
-    public ServerFormDetails(String formName, String downloadUrl, String manifestUrl, String formID,
-                             String formVersion, String hash,
-                             boolean isNotOnDevice, boolean isUpdated, ManifestFile manifest) {
+    public ServerFormDetails(String formName, String downloadUrl, String formID, String formVersion, String hash, boolean isNotOnDevice, boolean isUpdated, ManifestFile manifest) {
```

Comment:
We don't need a manifest url in here any longer as the manifest is fetched before creating this object.